server/kafka/manager.go

```diff
@@ -39,8 +40,16 @@ func NewManager(cc conf.ConnectorConfig, bc conf.NATSKafkaBridgeConfig) (Manager
 
 	if cc.SASL.User != "" {
 		sc.Net.SASL.Enable = true
+		sc.Net.SASL.Mechanism = sarama.SASLTypePlaintext
 		sc.Net.SASL.User = cc.SASL.User
 		sc.Net.SASL.Password = cc.SASL.Password
```

Comment:
`sc.Net.SASL.Handshake` not needed?

Reply:
It defaults to `true`.
https://pkg.go.dev/github.com/Shopify/sarama#Config

But we could add it to make sure it doesn't change beneath our feet.